stockAndPrices/services/app/main.py

```diff
@@ -70,19 +74,24 @@ def signUp():
         user = request.form.get('username')
         pwd = request.form.get('password')
         email = request.form.get('email')
-
+        val = validate(user,pwd,email,True)
+        if val != 'true':
```

Comment:
Since we are reusing the 'true' string, can we store this as constant somewhere?